radiation/python/radiation_aerosols.py

```diff
@@ -1513,22 +1514,46 @@ def setaer(
         #  ==================================================================  !
 
         #  ---  outputs:
-        aerosw = np.zeros((IMAX, NLAY, nbdsw, self.NF_AESW))
-        aerolw = np.zeros((IMAX, NLAY, NBDLW, self.NF_AELW))
+        # aerosw = np.zeros((IMAX, NLAY, nbdsw, self.NF_AESW))
+        # aerolw = np.zeros((IMAX, NLAY, NBDLW, self.NF_AELW))
 
         aerodp = np.zeros((IMAX, self.NSPC1))
```

Comment:
I kept this as a numpy array since this wasn't operating inside a stencil for the time being.  If it's in a stencil, I can imagine this as a "1D-multi-dimensional" storage of size [IMAX][self.NSPC1] where there will be a for loop over the extra dimension.